monkey/common/BUILD

```diff
@@ -1 +0,0 @@
-dev
```

Comment:
Why remove this? When running from source, it will say "+dev", which is preferable to just the version number. When build, with `build_package.sh`, this will get overwritten.